src/routes/restaurant.js

```diff
@@ -4,7 +4,7 @@ const { auth } = require('../utils/auth')
 
 router.route('/sign-up').post(restaurantController.signup);
 router.route('/sign-in').post(restaurantController.signin);
-router.route('/').get(restaurantController.list);
+router.route('/').get(auth, restaurantController.list);
```

Comment:
Este para que lo vas a implemetar?

Reply:
Es para que cuando le demos al link "restaurantes" del header. Traiga el listado de todos los restaurantes desde el BE. Pero debe estar logeado para que se muestren. Asi lo pense inicialmente pero podemos dejarlo sin ese requerimiento de autenticacion. que opinas?

Reply:
Yo creería que podemos dejarlo sin ese requerimiento, porque a la final esa navegación se presenta en el header cuando no esta logueado el usuario, lo que no puede hacer es acceder a cada restaurante luego.

Yo diría así.

Reply:
Listo estoy de acuerdo. cierro este pull entonces.